apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs)
+  : m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs)
+{
+  m_servoPwmOut.period_ms(20);
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees)
+{
+  if (degrees <= m_servoRangeInDegrees && degrees >= 0 && m_minPulsewidthInMs + degrees/m_servoRangeInDegrees <= m_maxPulsewidthInMs)
```

Comment:
You don't need to check the last condition, that's the mapping that your function is doing